modules/gitops-about-argo-rollout-manager-custom-resources-and-spec.adoc

```diff
@@ -0,0 +1,38 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: CONCEPT
+[id="gitops-about-argo-rollout-manager-custom-resources-and-spec_{context}"]
+= About RolloutManager custom resources and specification
+
+After you install {gitops-title} Operator on the cluster, to use the Argo Rollouts feature, you must create and submit RolloutManager custom resource (CR) with the required details to the {gitops-shortname} Operator in the namespace of your choice. But you can use Argo Rollouts in multiple namespaces as needed. The Operator creates an `argo-rollouts` instance with the following namespace-scoped supporting resources:
+
```

Comment:
@iam-veeramalla  I tried installing Argo Rollouts with 1.9 cluster. I am unable to see this CR - argo-rollouts-manager-metrics-reader (ClusterRole).

I see the CR mentioned here in this doc: https://docs.google.com/document/d/1QlXHOidAvLFz10PwOMWmzXqlipqHES9EJr_-RticHyY/edit#heading=h.9vauumaztq62

Should I mention the argo-rollouts-manager-metrics-reader (ClusterRole) in this list? Please let me know. Thanks!

Reply:
Not required. @iam-veeramalla just confirmed.